options/internal/generic/threads.cpp

```diff
@@ -93,7 +93,7 @@ static constexpr unsigned int mutex_owner_mask = (static_cast<uint32_t>(1) << 30
 static constexpr unsigned int mutex_waiters_bit = static_cast<uint32_t>(1) << 31;
 
 // Only valid for the internal __mlibc_m mutex of wrlocks.
-static constexpr unsigned int mutex_excl_bit = static_cast<uint32_t>(1) << 30;
+[[maybe_unused]] static constexpr unsigned int mutex_excl_bit = static_cast<uint32_t>(1) << 30;
```

Comment:
I think this shouldn't be here at all? It's also defined in `pthread-stubs.cpp` and used there.